src/devices/bus/msx/ctrl/ctrl.cpp

```diff
@@ -0,0 +1,43 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+/**********************************************************************
+
+    MSX General Purpose port emulation
+
+**********************************************************************/
+
+#include "emu.h"
+#include "ctrl.h"
+#include "joystick.h"
+#include "mouse.h"
+
+
+DEFINE_DEVICE_TYPE(MSX_GENERAL_PURPOSE_PORT, msx_general_purpose_port_device, "msx_general_purpose_port", "MSX General Purpose port")
+
+
+device_msx_general_purpose_port_interface::device_msx_general_purpose_port_interface(const machine_config &mconfig, device_t &device) :
+	device_interface(device, "vcsctrl")
```

Comment:
Should probably use something other than `"vcsctrl"` for this.

Reply:
Yes, silly leftover, changed it.